test/specs/utils/deepMerge.spec.js

```diff
@@ -26,15 +26,15 @@ describe('utils::deepMerge', function () {
 
   it('should deepMerge recursively', function () {
     var a = {foo: {bar: 123}};
-    var b = {foo: {baz: 456}, bar: {qux: 789}};
+    var b = {foo: {baz: 456}, bar: {qux: null}};
```

Comment:
It's better to **add** tests, instead of reusing/modifying.

Reply:
I agree, but in this specific case the modification does not remove any assertion from the test (integer values inside the object are still tested via `baz: 456`), so I feel like this modification would be okay.